Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/core/impl/ConceptManager.java

```diff
@@ -66,7 +66,7 @@
     @Override
     public ConceptEntry getConceptEntry(String id) {
         ConceptEntry entry = client.getEntry(id);
-        if (entry != null) {
+        if (entry != null && !entry.isDeleted()) {
```

Comment:
I have my reservations against this approach. The system is built on the assumption that if you give this method a valid id, it'll return a concept (e.g. for synonyms). If you change this it'll have unexpected side effects (e.g. when retrieving synonyms for a concept). If you want to make sure the search doesn't return a deleted concept, you should do that differently. Also, when you said "searching" for a concept, I thought you meant actually searching for the id through the search interface. But you meant retrieving a concept by its ID, right? In this case, CP should probably return a concept but say that it is a deleted concept. Otherwise applications that rely on the deleted concept would break.